src/CodeStyle/Core/CodeFixes/FormattingCodeFixProvider.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic, Name = "FixFormatting")]
+    [Shared]
+    internal class FormattingCodeFixProvider : CodeFixProvider
+    {
+        public override ImmutableArray<string> FixableDiagnosticIds
+            => ImmutableArray.Create(AbstractFormattingAnalyzer.FormattingDiagnosticId);
+
+        public override FixAllProvider GetFixAllProvider()
+        {
+            return new FixAll();
```

Comment:
i don't like the nameof this type.

Reply:
💭 `FormattingCodeFixProvider.FormatDocumentFixAllProvider` seemed unnecessarily verbose